Makefile

```diff
@@ -21,7 +21,11 @@ lint: node_modules/.uptodate
 
 .PHONY: docs
 docs:
-	cd docs && make livehtml
+	tox -e py3-docs
```

Comment:
For the client I'm using `py3` instead of `py36` since it's only ever used for building docs.